.eslintrc.cjs

```diff
@@ -97,7 +97,7 @@ module.exports = {
         ],
         '@typescript-eslint/prefer-optional-chain': 'warn',
         'camelcase': 'off',
-        'max-len': ['warn', { code: 120, comments: null }],
```

Comment:
I only added this for comments lol, removing as this is invalid, and we already have prettier 😄